package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "oss-bot",
+  "description": "Github monitoring robot.",
+  "dependencies": {
+    "@types/marked": "0.0.28",
+    "diff": "^3.2.0",
+    "firebase-admin": "^4.1.2",
+    "firebase-functions": "^0.5",
+    "github": "^9.2.0",
+    "mailgun-js": "^0.8.2",
+    "marked": "^0.3.6",
+    "mocha": "^3.3.0",
+    "request": "^2.81.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.0.1",
+    "@types/diff": "^3.2.0",
+    "@types/request": "0.0.45",
+    "jest": "^20.0.4",
+    "ts-jest": "^19.0.14",
+    "tslint": "^5.0.0",
+    "typescript": "^2.4.1"
+  },
+  "scripts": {
+    "test": "mocha",
+    "debug": "mocha --debug-brk",
+    "fmt": "prettier --single-quote --write \"{test/,}*.js\"",
```

Comment:
Actually you can use prettier on ts as of v1.4:
https://github.com/prettier/prettier/releases/tag/1.4.0

So let's just configure that.